test/csx.coffee

```diff
@@ -230,9 +230,9 @@ test 'nested CSX within an attribute, with object attr value', ->
   ''', '''
     <Company>
       <Person name={<NameComponent attr3={{
-          'a': {},
-          b: '{'
-        }} />} />
+      'a': {},
+      b: '{'
+    }} />} />
```

Comment:
Can you please explain why the indentation levels changed in these tests?

Reply:
Looks like it's because there's no longer the
```
idt = o.indent += TAB
```
which was set before calling `@compileCSXAttributes()` in `Obj::compileNode()`

Either way we wouldn't have consistent indentation of multiline JSX attribute values (with respect to their enclosing JSX element) since nested JSX elements aren't indented in a structured way when printing (we're just preserving whitespace), but these are mostly an improvement